packages/ts-transform-graphql-js-tag/src/index.ts

```diff
@@ -0,0 +1,201 @@
+import ts from "typescript";
+import {
+  DefinitionNode,
+  FragmentDefinitionNode,
+  OperationDefinitionNode,
+  parse,
+  Kind,
+} from "graphql";
+import { collapseTextChangeRangesAcrossMultipleVersions } from "relay-compiler/node_modules/typescript";
+// import astify, { InterpolationNode } from "./astify";
+
+export interface GraphQLTagTransformOptions {
+  graphqlTagModuleRegex: RegExp;
+}
+
+const DefaultOptions: GraphQLTagTransformOptions = {
+  graphqlTagModuleRegex: new RegExp(/^['"]@graphitation\/graphql-js-tag['"]$/),
+};
+
+export function getTransformer(
+  options: Partial<GraphQLTagTransformOptions>
+): ts.TransformerFactory<ts.SourceFile> {
+  const fullOptions: GraphQLTagTransformOptions = {
+    ...DefaultOptions,
+    ...options,
+  };
+
+  return (context: ts.TransformationContext): ts.Transformer<ts.SourceFile> => {
+    return (sourceFile: ts.SourceFile) =>
+      ts.visitNode(sourceFile, getVisitor(fullOptions, context, sourceFile));
+  };
+}
+
+function getVisitor(
+  options: GraphQLTagTransformOptions,
+  context: ts.TransformationContext,
+  sourceFile: ts.SourceFile
+): ts.Visitor {
+  const visitor: ts.Visitor = (node: ts.Node): ts.VisitResult<ts.Node> => {
+    // `graphql-tag` import declaration detected
+    if (ts.isImportDeclaration(node)) {
+      const moduleName = (node as ts.ImportDeclaration).moduleSpecifier.getText(
+        sourceFile
+      );
+      if (options.graphqlTagModuleRegex.test(moduleName)) {
+        // delete it
+        return undefined;
+      }
+    }
+
+    // tagged template expression detected
+    if (ts.isTaggedTemplateExpression(node)) {
+      const [tag, template] = node.getChildren();
+
+      const isTemplateExpression = ts.isTemplateExpression(template);
+      const isTemplateLiteral = ts.isNoSubstitutionTemplateLiteral(template);
+
+      if (
+        tag.getText() === "gql" &&
```

Comment:
relay-compiler looks for a `graphql` tag instead of `gql`

```suggestion
        tag.getText() === "graphql" &&
```

Reply:
There is an option now to customize it.